functions/src/mappers/github/status.mapper.ts

```diff
@@ -12,8 +12,8 @@ export interface GitHubPullRequestStatusModel {
   id: number;
   state: string;
   context: string;
-  createdAt: firestore.Timestamp;
-  updatedAt: firestore.Timestamp;
+  createdAt: string;
+  updatedAt: string;
```

Comment:
These should not be strings for firestore

Reply:
Because we are not saving these fields in firestore database and when I am trying to get the time from timestamp it was giving some error.

Reply:
Do you remember the error?

Reply:
I think we will eventually be saving these fields

Reply:
I think the only difference with other timestamp fields, I found is these values are not getting saved in DB and others are.
We are retrieving the other values from collection not from API directly
Here it is coming as timestamp object in existing ones. Please see the attached screenshots

![image (2)](https://user-images.githubusercontent.com/10898595/67227284-940fbb80-f454-11e9-8e44-10d39f993806.png)

![image (3)](https://user-images.githubusercontent.com/10898595/67227286-940fbb80-f454-11e9-9254-d16cfa256594.png)

Reply:
This will be updated back from `string` to `firestore.Timestamp` in #1556